aicsimageio/formats.py

```diff
@@ -18,359 +18,460 @@
 # formats, but TiffReader does as well and it has better
 # dask chunking + metadata parsing than DefaultReader for those formats
 
+# BASE-IMAGEIO FORMATS (with tifffile + non-existant removals)
+#
+# Pulled using:
+# from imageio import formats
+# routes = {}
+# for f in formats:
+#   for ext in f.extensions:
+#       routes[ext[1:]] = ["aicsimageio.readers.default_reader.DefaultReader"]
+
+FORMAT_IMPLEMENTATIONS: Dict[str, List[str]] = {
```

Comment:
there is something vaguely horrible about this. 
Apart from restructuring everything, what do you think of putting it in alphabetical order by key?

Reply:
alphabetized :+1: